# HG changeset patch
# User John W. Eaton <jwe@octave.org>
# Date 1202878369 18000
#      Tue Feb 12 23:52:49 2008 -0500
# Branch release-3-0-x
# Node ID cdb52e2ead2c7e39c135e7b6d3f8190f0599c18d
# Parent  ce1da7a951dc310f2aadeca8ca791011f5413132
add Jason Riedy to contributors file

diff --git a/doc/interpreter/contributors.in b/doc/interpreter/contributors.in
--- a/doc/interpreter/contributors.in
+++ b/doc/interpreter/contributors.in
@@ -145,16 +145,17 @@ Tom Poage
 Orion Poplawski
 Ondrej Popp
 Jef Poskanzer
 Francesco Potorti
 James B. Rawlings
 Eric S. Raymond
 Balint Reczey
 Michael Reifenberger
+Jason Riedy
 Petter Risholm
 Matthew W. Roberts
 Andrew Ross
 Mark van Rossum
 Kevin Ruland
 Olli Saarela
 Toni Saarela
 Juhani Saastamoinen
